### PR TITLE
travelSafely

### PR DESCRIPTION
**Pull request by chobobobo**
stop travelling at range 1 if the destination is blocked by a creep

### CODE REVIEW

**Comment by chobobobo** [comment]
this is to resolve https://github.com/ScreepsOCS/screeps.behaviour-action-pattern/issues/1177

File: creep.action.travelling.js [hunk 1]
```diff
@@ -27,6 +27,12 @@ action.step = function(creep){
         const range = creep.pos.getRangeTo(target);
         if( range <= targetRange ) {
             return action.unregister(creep);
+        } else if (targetRange === 0 && creep.pos.isNearTo(target)) {
+            if (target.pos.look().filter({type: 'creep'}).length > 0) {
```
Comment (Spedwards) [inline creep.action.travelling.js]: `target.pos.lookFor(LOOK_CREEPS)`

File: creep.action.travelling.js [hunk 2]
```diff
@@ -27,6 +27,12 @@ action.step = function(creep){
         const range = creep.pos.getRangeTo(target);
         if( range <= targetRange ) {
             return action.unregister(creep);
+        } else if (targetRange === 0 && creep.pos.isNearTo(target)) {
```
Comment (Spedwards) [inline creep.action.travelling.js]: You said in the PR description, "target range 1"

Comment (chobobobo) [inline creep.action.travelling.js]: yes, stop travelling at range 1 if you are blocked, this checks to see if you are trying to go to range 0 and adjacent to your target (range 1)

File: creep.action.travelling.js (hunk 1 above)
Comment (chobobobo) [inline creep.action.travelling.js]: good catch

File: creep.action.travelling.js (hunk 2 above)
Comment (Spedwards) [inline creep.action.travelling.js]: Ah okay. Makes sense.

**Pull request merged**